[4.0.0] Add gateway environment configuration for enforcer and router (#5897)
wso2/docs-apim: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md

Let's assume you have two environments called "choreo-connect-1" and "choreo-connect-2".

!!! Important

>     Even though the adapter has the capability to deploy APIs under multiple environment Labels (gateway labels), it is recommanded to apply for a single gateway environment label to maintain the deployment architecture simple.

Reviewer: Check if this is ok @renuka-fernando

> 1. In each of the Adapter, add the relevant environment to `environmentLabels` in `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect-with-apim/conf/config.toml` under `controlPlane`.

Reviewer: ```suggestion
1. In each of the Adapters, add the relevant environment to `environmentLabels` in `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect-with-apim/conf/config.toml` under `controlPlane`.
```

        reconnectInterval = 5000
        reconnectRetryCount = 60
    ```

> 2. Next let's configure the Gateway environment label in the Enforcer and Router. For that update the following environment variables in each service in the docker-compose file.

Reviewer: ```suggestion
2. Next let's configure the Gateway environment label in the Enforcer and Router. To do that, update the following environment variables in each service in the docker-compose file.
```

> 2. Configure gateway environment label in Enforcer and Router. Update the following environment labels in each component. Update the following environment variables defined in the docker-compose file.

Reviewer: ```suggestion
2. Configure the Gateway environment label in the Enforcer and Router. Update the following environment labels in each component. Update the following environment variables defined in the docker-compose file.
```

Reviewer: There are too many steps lumped in together here. Can we break this into multiple steps or reword this to make sure people know what they are doing here?

Author: ```suggestion
2. Next let's configure the Gateway environment label in the Enforcer and Router. For that update the following environment variables in each service in the docker-compose file.
```
Yes we can keep this as a one step, reworded @samgnaniah could you please check this again?

   
   -  Enforcer
      ``` tab="Format"
      ENFORCER_LABEL=<GATEWAY_LABEL>
      ```
      ``` tab="Enforcer 1"
      ENFORCER_LABEL=choreo-connect-1
      ```
      ``` tab="Enforcer 2"
      ENFORCER_LABEL=choreo-connect-2
      ```

   -  Router
      ``` tab="Format"
      ROUTER_LABEL=<GATEWAY_LABEL>
      ```
      ``` tab="Enforcer 1"
      ROUTER_LABEL=choreo-connect-1
      ```
      ``` tab="Enforcer 2"
      ROUTER_LABEL=choreo-connect-2
      ```

3. Start Choreo Connect deployment from docker-compose.

> 4. Add the Gateway environments "choreo-connect-1" and "choreo-connect-2" from the Gateways in the APIM admin portal.

Reviewer: Are these environments or labels?

Author: environment labels, but in the admin portal below it is gateway environments
